contributors.js

```diff
@@ -63,5 +63,7 @@ const contributors = [
   { username: 'anushkabhansalii', message: 'Keep smiling!' },
   { username: 'sohailshk', message: 'First pr hopefully for hacktoberfest2024' },
   { username: 'snehachandra1108', message: 'First Hacktober' },
-  { username: 'its-wasp', message: 'I\'m the hecker boi'}
+  { username: 'its-wasp', message: 'I\'m the hecker boi'},
+  { username: 'Karthikeya914', message: 'Duck Duck' },
+  { username: 'Karthikeya914', message: 'Duck Quack' },
```

Comment:
Hello @Karthikeya914 , I am sorry but you have 2 lines for your duck. Please remove one 🙂

Reply:
removed one kindly please check

Reply:
hello @jeanphi-baconnias , i have removed a line kindly check and also correct me if i am wrong 🙂